core/src/js/utilities/events.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Create an event with the specified name in a browser-agnostic way.
+ * @param {string} eventName - the name of the event
```

Comment:
This parameter is not {string} bound but rather mixed as it accepts strings or functions.

Reply:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event), the first parameter to `new Event()` is a DOMString, and "[DOMString is mapped directly to a String](https://developer.mozilla.org/en-US/docs/Web/API/DOMString)." Also, my wrapper function explicitly test for string, and fails if it's not a (non-empty) string.

Reply:
Oh, I totally read the if statements wrong. I thought the second check where the if statement is checking to see if Event is a function was on the input parameter. You are 100% right. Apologies for this comment.